mod/metrics/lib/card/vega_chart/range_chart.rb

```diff
@@ -41,29 +40,16 @@ def data_item_hash filter, _count
       end
 
       def x_axis
-        # super.merge scale: "x_label", title: title_with_unit("Ranges")
-        super.merge format: "~s", title: title_with_unit("Ranges")
+        super.merge scale: "x_label", format: "~s", title: title_with_unit("Ranges")
       end
 
-      # def scales
-      #   super << x_label_scale
-      # end
-      #
-      # def x_label_scale
-      #   { name: "x_label", type: "point", range: "width", domain: x_labels }
-      # end
-      #
-      # def x_labels
-      #   precision = 3
-      #   while precision < 10
-      #     human = @labels.map do |num|
-      #       @format.number_to_human num.to_f, precision: precision
-      #     end
-      #
-      #     return human if human.uniq.size == human.size
-      #     precision += 1
```

Comment:
What's the problem with this?

Reply:
The old issue was that when used with logarithmic scales, it sometimes produced non-unique labels, and for whatever reason, vega reduced it to the uniques.

The above fixed it, but `format: "~s"` fixed it much more easily.